static/css/style.css

```diff
@@ -164,83 +163,71 @@ footer ul li a {
     border-color: var(--p-orange);
 }
 
-/* Landing Page */
-
-.landing-page {
-    margin: 10px 10px 30px;
-    align-content: center;
-    justify-items: center;
+.divider-orange{
+    background-image: radial-gradient(var(--p-orange), var(--p-red));
+    height: 4px;
+    border-radius: 100%;
 }
 
-.drk-txt-hkth {
-    text-shadow: 1px 1px 2px var(--p-grey);
+.delete-resource-button {
+    cursor: pointer;
+}
+/* EDIT PROFILE */
+.edit_form {
+    margin: auto;
+    padding: 2.5vh;
+    width: max-content;
+    margin-bottom: 5vh;
+    border: 1px solid #E84610;
 }
 
-.card-header {
-    background-color: var(--t-orange);
+.edit_title, .edit_form{
+    text-align: center;
 }
 
-.card-header .btn-link {
-    text-decoration: none;
+.edit_title {
     color: var(--p-grey);
-    font-size: 1.15rem;
-    transition: .3s;
 }
-
-.card-header .btn-link:hover {
-    cursor: pointer;
-    color: var(--p-orange);
+.edit_form label {
+    width: 100%;
 }
 
-
-#accordion-wrapper>h1,
-#carousel-wrapper>h1 {
-    font-size: 2rem;
-    margin-bottom: 20px;
-    color:var(--black);
+.edit_form label, .edit_title, .edit_form button {
+    padding: 1vh;
 }
 
-.project-list>ol{
-     margin: 20px;
-     font-size: 1.7rem;
-     text-align: left;
-     color: var(--black);
-     padding: 6px 6px;
-
+.edit_form select, .edit_form {
+    max-width: 100%;
 }
 
-.project-list {
-    width: 100%;
-    height: auto;
-    background-color:var(--t-orange);
-    border-radius: 10px;
+.edit_form button {
+    margin-top: 2vh;
+    background-color: var(--p-orange);
+    border-color: var(--p-grey);
+    color: var(--s-grey);
 }
 
-.list li{
-    font-size: 0.5em;
-    text-align: left;
-
+.edit_form button:hover {
+    color: var(--p-grey);
 }
-
-@media (max-width:453px) {
-    #accordion-wrapper>h1,
-    #carousel-wrapper>h1 {
-        font-size: 1.5rem;
-
-    }
+.bg-p-blue {
+    background-color: var(--p-blue);  
 }
 
-@media (max-width:993px) {
-    #carousel-wrapper {
-        margin-top: 30px;
-    }
-    
+/* TOASTS */
+/*This allows for multiple toasts to stack.*/
+.toast-wrapper {
+    position: absolute;
+    top: 0;
+    right: 0;
+    min-width: 300px;
 }
 
-@media (max-width:993px) {
-    
-    .card-header h2 .btn-link {
-    color:var(--black);
-    font-size: 1.1rem;
-}
+/*This positions the toast on the page.*/
+.toast-position {
+    position: fixed;
+    top: 80px;
+    right: 20px;
+    min-height: 200px;
+    z-index: 1021;
 }
```

Comment:
Needs a blank line at the end of the file.

Reply:
done